.github/workflows/main.yml

```diff
@@ -75,6 +80,60 @@ jobs:
             !**/.env/**
             !**/node_modules/**
             !**/project/.m2/**
+
+  create-release-package:
+    name: Create Release Package
+    runs-on: ubuntu-latest
+    steps:
+      # Set up all of our standard runtimes
+      - name: Set up .NET 3.1
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '3.1.x'
+      - name: Set up Java 8
+        uses: actions/setup-java@v1
+        with:
+          java-version: '8'
+      - name: Set up Node 12
+        uses: actions/setup-node@v2.1.1
+        with:
+          node-version: '12'
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
+      - name: 'Linux: Install python3-venv'
+        if: runner.os == 'Linux'
+        run: sudo apt install -y python3-venv
+      - name: 'Windows: Expose python3 command'
+        if: runner.os == 'Windows'
+        shell: bash
+        run: cp ${pythonLocation}/python.exe ${pythonLocation}/python3.exe
```

Comment:
When will this evaluate to true, given this all "runs-on" ubuntu-latest?

Reply:
You're correct. I've kinda standardized on using the same setup process everywhere, but this never runs on Windows, so I should probably just remove that...